lib/util/miq-password.rb

```diff
@@ -121,21 +121,18 @@ def self.v2_key
     @v2_key ||= ez_load("#{key_root}/v2_key")
   end
 
-  def self.v1_key
-    @v1_key ||= ez_load("#{key_root}/v1_key")
-  end
-
-  def self.v0_key
-    @v0_key ||= File.exist?("#{key_root}/v0_key") &&
-      begin
-        params = YAML.load_file("#{key_root}/v0_key")
-        CryptString.new(nil, params[:algorithm], params[:key], params[:iv])
-      end
+  def self.add_key(filename, type = :v1)
```

Comment:
This is awesome @kbrock.   Two questions :+1: 

- wondering if you should name this method add_legacy_key or something to show that it really should not be used. 
- should type be optional? Or should the caller be forced to specify? 
- should it be the value :v1/:v0 or just a numeric 0/1 and name the variable version or something?